charts/mangos/templates/secret.yaml

```diff
@@ -0,0 +1,26 @@
+{{- define "secret.login" }}
+    {{ template "chart.fullname" . }}-mysql-service;3306;root;{{ template "getPassword" . }};realmd
+{{- end }}
+
+{{- define "secret.world" }}
+    {{ template "chart.fullname" . }}-mysql-service;3306;root;{{ template "getPassword" . }};mangos{{ .Values.wowVersion }}
```

Comment:
For mangos 2 & 3, databases are "mangos" and "characters"

Reply:
You're right, I updated it to address this